apps/book/schema.py

```diff
@@ -68,8 +71,12 @@ class Meta:
     og_image = graphene.Field(FileFieldType)
 
     @staticmethod
-    def resolve_quantity_in_cart(root, info, **kwargs) -> int:
-        return info.context.dl.book.quantity_in_cart.load(root.pk)
+    def resolve_cart_details(root, info, **kwargs) -> QuerySet:
+        return root.book_cart_item.first()
```

Comment:
Why no data loader? N+1 issue.

Reply:
@thenav56  Added data loader

Reply:
N+1 issue is still in the data loader.

Reply:
@thenav56  We should pass cart item object from dataloader, do you have better solution